lib/core/constants/image_paths/image_paths.dart

```diff
@@ -78,6 +78,7 @@ class PngImagePaths {
   static const String step2Img = 'assets/images/step2.png';
   static const String step31Img = 'assets/images/step3.png';
   static const String testatorImg = 'assets/images/testator.png';
+  static const String buildImg = 'assets/images/build.png';
```

Comment:
follow the image and icons name format(mentioned above)
for image : imgCar = ....
for icon icArrow = ....